lib/internal/Magento/Framework/Css/PreProcessor/File/Collector/Aggregated.php

```diff
@@ -6,6 +6,7 @@
 
 namespace Magento\Framework\Css\PreProcessor\File\Collector;
 
+use Magento\Framework\Module\ModuleManagerInterface;
```

Comment:
This Interface does not exist you should use Magento\Framework\Module\Manager instead

Reply:
Good catch! Could you fix it?